session/pingpong/invoice_messaging.go

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package pingpong
+
+import (
+	"github.com/mysteriumnetwork/node/communication"
+	"github.com/mysteriumnetwork/payments/crypto"
+)
+
+// InvoiceRequest structure represents the invoice message that the provider sends to the consumer
+type InvoiceRequest struct {
+	Invoice crypto.Invoice `json:"invoice"`
+}
+
+const endpointInvoice = "session-invoice"
+const messageEndpointInvoice = communication.MessageEndpoint(endpointInvoice)
```

Comment:
I see where this is coming from, but let's use normalized order for naming, it makes it easier to comprehend:
- `endpointInvoice` → `invoiceEndpoint`
- `messageEndpointInvoice` → `invoiceMessageEndpoint`